src/status_im/ui/screens/discover/styles.cljs

```diff
@@ -37,14 +39,21 @@
 (def carousel-page-style
   {})
 
-(def tag-name
+(def tag-button
   {:color           color-blue
    :font-size       14
    :padding-right   5
    :padding-bottom  2
    :align-items     :center
    :justify-content :center})
 
+(def tag-name
+  {:color            color-blue
+   :background-color :white
```

Comment:
Is this how we refer to `white`? If we have a `color-white` it is better to use that to be consistent imo

Reply:
Re: palette confirmation